group/templates/group/viewall.html

```diff
@@ -97,5 +81,18 @@ <h4 class="modal-title">Edit Annowence</h4>
       {% endfor %}
     </tbody>
   </table>
+  <nav>
+    <ul class="pager">
```

Comment:
You can include ```pager.html``` in [utils](https://github.com/bruce3557/NTHUOJ_web/blob/dev/utils/templates/utils/pager.html)